.github/workflows/build.yml

```diff
@@ -54,11 +54,41 @@ jobs:
             org.opencontainers.image.description="RomM (ROM Manager) allows you to scan, enrich, and browse your game collection with a clean and responsive interface. With support for multiple platforms, various naming schemes and custom tags, RomM is a must-have for anyone who plays on emulators."
             org.opencontainers.image.licenses="AGPLv3"
 
+      - name: Generate Docker metadata (slim)
+        id: meta-slim
+        uses: docker/metadata-action@v5
+        with:
+          images: |
+            name=rommapp/romm
+            name=ghcr.io/rommapp/romm
+          tags: |
+            type=semver,pattern={{version}}-slim
+            type=semver,pattern={{major}}.{{minor}}-slim
+            type=semver,pattern={{major}}-slim
+          labels: |
+            org.opencontainers.image.version={{version}}-slim
+            org.opencontainers.image.title="rommapp/romm"
+            org.opencontainers.image.description="RomM (ROM Manager) allows you to scan, enrich, and browse your game collection with a clean and responsive interface. With support for multiple platforms, various naming schemes and custom tags, RomM is a must-have for anyone who plays on emulators."
+            org.opencontainers.image.licenses="AGPLv3"
+
       - name: Set version
         run: |
           sed -i 's/<version>/${{ steps.meta.outputs.version }}/' backend/__version__.py
 
-      - name: Build base image
+      - name: Build full image
```

Comment:
I think we should build `slim` first and `full` afterwards. With that configured, we can check if the CI is correctly caching previous stages.